services/spar/test-integration/Util.hs

```diff
@@ -0,0 +1,109 @@
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE QuasiQuotes         #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE ViewPatterns        #-}
+
+-- | TODO: this is all copied from /services/galley/test/integration/API/Util.hs and some other
+-- places; should we make this a new library?
```

Comment:
Actually, I would like to do that in this PR, any objections?  (I wouldn't touch brig, that can be done separately.)

Reply:
No, that's fine